packages/paste-core/components/input/src/Input.tsx

```diff
@@ -4,7 +4,7 @@ import {styled, css} from '@twilio-paste/styling-library';
 import {InputBox} from '@twilio-paste/input-box';
 import {safelySpreadFormControlProps} from './utils';
 
-export type InputTypes = 'text' | 'email' | 'hidden' | 'number' | 'password' | 'search' | 'tel' | 'date';
+export type InputTypes = 'text' | 'email' | 'hidden' | 'number' | 'password' | 'search' | 'tel' | 'date' | 'time';
```

Comment:
Could these be imported from input-box? That would save some duplication.

Reply:
Good catch! Done